desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -255,6 +260,8 @@ public void onRemove(final InputColumn<?> sourceColumn) {
 
     private static final Logger logger = LoggerFactory.getLogger(AnalysisJobBuilderWindow.class);
     private static final ImageManager imageManager = ImageManager.get();
+    
+    public static final List<Func<AnalysisJobBuilderWindowImpl, JComponent>> PLUGGABLE_LABELS_BANNER_COMPONENTS = new ArrayList<>(0);
```

Comment:
Not sure I understand? Using the ServiceLoader, you would provide an interface that any JAR could then implement. Then DataCleaner would just load all available implementations, and add them by itself.

For inspiration, see how we provide an implementation of SparkJobLifeCycleListener in the commercial spark package, and add it in `org.datacleaner.spark.Main` line 67.

Of course, this being a Swing class, the actual service needs to be some kind of factory that provides the `PluggablePanel` in `getPluggablePanel()` or similar method.